src/main/resources/attackvectors/BlindSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,29 @@
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_1=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending true and false statements.<br/>\
+  How to exploit this level?<br/>\
+  Insert those in textbox<br/>\
+  For true statement<br/>\
+  <i>100 OR 2=2</i><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": true }</i><br/>\
+We know <i>100 id</i> is not present in the table but still, it is returning <i>true</i>. it means that the input is evaluated to true and the backend has returned that car is present, which could indicate vulnerability.<br/>\
+  For false statement<br/>\
+  <i>1 AND 1=2</i><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": false }</i> <br/>\
+  We know that car with <i>id = 1</i> is present but because of second part backend returns <i>false</i>.
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_2=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending true and false statements.<br/>\
+  How to exploit this level?<br/>\
+  Previous queries <i>1 OR 2=2</i> and <i>1 AND 1=2</i> return <i>System Error Occurred. please check logs</i>.<br/>\
```

Comment:
Please don't refer previous query instead tell the query.

Reply:
You mean, I should remove line
Previous queries <i>1 OR 2=2</i> and <i>1 AND 1=2</i> return <i>System Error Occurred. please check logs</i>. 
or just leave
<i>1 OR 2=2</i> and <i>1 AND 1=2</i> return <i>System Error Occurred. please check logs</i>.

Reply:
May be something like: 
```Following input: 1 OR 2=2 and 1 AND 1=2 returns System Error Occurred but doesn't give any details like table name or queries etc.` Actually additional security was added ...```